R/exportCohortDefinitionSet.R

```diff
@@ -0,0 +1,57 @@
+#' Export cohort definition set from WebAPI
+#'
+#' @param baseUrl                 The base URL for the WebApi instance, for example:
+#'                                "http://server.org:80/WebAPI".
+#' @param cohortIds               A set of cohortIds to fetch from WebApi.
+#' @param generateStats           Should cohort inclusion rule statistics be generated?
+#'
+#' @details
+#' Constructs a CohortDefinition set containing the following fields: \describe{
+#' \item{atlasId}{The cohort ID in
+#' ATLAS.} \item{cohortId}{a copy of the value in atlasId.} \item{cohortName}{The name of the cohort.}
+#' \item{sql}{The cohort generation sql.} \item{json}{The cohort definition JSON.}
+#' \item{logicDescription}{The cohort description.}}
+#'
+#' @export
+exportCohortDefinitionSet <-
+  function(baseUrl, cohortIds, generateStats = FALSE) {
+    if (length(cohortIds) == 0) {
+      stop("Must provide a non-zero length cohortIds vector.")
+    }
+    cohortDefinitionSet <- dplyr::tibble(
+      atlasId = integer(),
+      cohortId = integer(),
+      cohortName = character(),
+      sql = character(),
+      json = character(),
+      logicDescription = character(),
+      generateStats = logical()
+    )
+    
+    for (i in (1:length(cohortIds))) {
+      cohortId <- cohortIds[i]
+      ParallelLogger::logInfo(paste("Fetching cohortId:", cohortId))
```

Comment:
As stated above, I would prefer a call to `message` here to remove the dependency.